RUFAS/output_manager.py

```diff
@@ -328,7 +328,21 @@ def _dict_to_csv_column_list(self, data_dict: Dict[str, List[Any]], mandatory_fi
         """
         column_list = []
         for field in mandatory_fields:
-            column_list.append((field, pd.Series(data_dict[field], dtype=object)))
+            data_list = data_dict[field]
+            if len(data_list) != 0 and isinstance(data_list[0], dict):
```

Comment:
I think this could be shortened slightly:
```suggestion
            if data_list and isinstance(data_list[0], dict):
```